lib/entities/scheduled-action.ts

```diff
@@ -101,19 +101,15 @@ export type CreateUpdateScheduledActionProps = Pick<
   'action' | 'entity' | 'environment' | 'scheduledFor' | 'payload'
 >
 
-export interface ScheduledActionCollection {
-  sys: {
-    type: 'Array'
-  }
-  pages: BasicCursorPaginationOptions
-  limit: number
-  items: ScheduledActionProps[]
-}
+export type ScheduledActionCollection = CursorPaginatedCollectionProp<ScheduledActionProps>
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
-export interface ScheduledActionQueryOptions extends BasicCursorPaginationOptions {
+export interface ScheduledActionQueryOptions {
   'environment.sys.id': string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   [key: string]: any
+  limit?: number
+  next?: string
+  prev?: string
```

Comment:
Not sure if that's right.

The response from an api supporting cursor based pagination should be

```
pages: {
  next: foo,
  prev: bar,
}
```

and passed to a query it should be `pageNext` or `pagePrev`

Here the typing is called `ScheduledActionQueryOptions` - so it should be `pageNext` or `pagePrev` because it's a query right?

Reply:
For me it looked like scheduled actions is using `next` and `prev` as url parameters to iterate through pages. I didn't want to break their interface because it was implemented with `next` and `prev` before.

Reply:
Understood 👍